examples/prerecorded_from_url.rs

```diff
@@ -0,0 +1,20 @@
+use deepgram::{Deepgram, DeepgramError, Language, OptionsBuilder, UrlSource};
+
+const DEEPGRAM_API_KEY: &str = "YOUR_SECRET";
```

Comment:
Examples should be runnable by users who download the SDK.  For this, maybe consider using an environment variable instead of a const in the main function.

Side note:  For string literals, `static` is probably a more usual choice than `const`.  A const gets copied into every location it gets used, while a `static` gets assigned a location in the binary's read-only memory section.

Reply:
Implemented in:
 * c24042cac528cb32091de10d69b4493a64983adb
 * 71200b650b6fb1ac6a0aa4d7af7c25ac13e65b85